crates/common/flockfile/src/unix.rs

```diff
@@ -83,13 +85,22 @@ impl Flockfile {
                     source: err,
                 })?;
 
+        // Convert the PID to a string
+        let pid_string = format!("{}", Pid::this());
+
         flock(file.as_raw_fd(), FlockArg::LockExclusiveNonblock).map_err(|err| {
             FlockfileError::FromNix {
                 path: path.clone(),
                 source: err,
             }
         })?;
 
+        // Write the PID to the lock file
+        write(file.as_raw_fd(), pid_string.as_bytes()).map_err(|err| FlockfileError::FromNix {
```

Comment:
You can combine the line 89 and this line together.

So remove this,
```
let pid_string = format!("{}", Pid::this());
```

Then,
```suggestion
        write(file.as_raw_fd(), Pid::this().to_string().as_bytes()).map_err(|err| FlockfileError::FromNix {
```